bouncer/views.py

```diff
@@ -83,7 +90,11 @@ def annotation(self):
                 "viaUrl": via_url,
                 "extensionUrl": extension_url,
             }),
-            "pretty_url": pretty_url
+            "pretty_url": pretty_url,
+            "quote": quote,
+            "text": text,
+            "shared": shared,
+            "group": group
```

Comment:
There are no tests ensuring that we return the correct values for `quote`, `text`, `shared`, and `group`, but we do have tests for all the other values.